docs/chart_best_practices/templates.md

```diff
@@ -132,6 +132,43 @@ metadata:
 
 ```
 
+## Resource Naming in Templates
+
+Hard-coding the `name:` into a resource is usually considered to be bad practice.
+Names should be unique to a release. So we might want to generate a name field
+by inserting the release name - for example:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-myservice
+```
+
+Or if there is only one resource of this kind then we could use .Release.Name or the template fullname function defined in _helpers.tpl (which uses release name):
```

Comment:
because of the underscore in _helpers.tpl, it's italicizing the rest of the document in the diff. Can you add a backslash to counteract markdown's parsing of the underscore?

Reply:
Thanks @bacongobbler , fixed this now